packages/nextjs/src/server/authMiddleware.ts

```diff
@@ -131,6 +131,10 @@ type AuthMiddlewareParams = WithAuthOptions & {
    * @default undefined
    */
   apiRoutes?: ApiRoutesParam;
+  /**
+   * A function that transforms the 401 Unauthorized response returned by the middleware when hitting an API endpoint.
+   */
```

Comment:
🙃 Maybe add a reminder here that this will be used only for routes marked as `apiRoutes`